src/main/java/com/southernartsoap/repository/CartDetailsRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.southernartsoap.repository;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+import com.southernartsoap.model.CartDetails;
+
+public interface CartDetailsRepository extends CrudRepository<CartDetails, Long> {
+	List<CartDetails> findAll();
+}
```

Comment:
should this be findAllByUserId(Long id)?

What happens if multiple people are logged in?